basics/uri.adoc

```diff
@@ -277,10 +277,10 @@ impl UUri {
 A UUri can be used to define a _pattern_ that other UUris can then be matched against.
 For that purpose, a UUri
 
-* *MAY* have its `authority_name` set to the `*` (`U+002A`, Asterisk) character in order to match _any_ (including _no_) authority,
-* *MAY* have the _service ID_ part of its `ue_id` set to `0xFFFF` in order to match _any_ type,
-* *MAY* have the _service instance ID_ part of its `ue_id` set to `0x0000` in order to match _any_ instance,
-* *MAY* have its `ue_version_major` set to `0xFF` in order to match _any_ version,
+* *MAY* have its `authority_name` set to the `*` (`U+002A`, Asterisk) character in order to match _any_ (including _no_) authority.
+* *MAY* have the _service ID_ part of its `ue_id` set to `0xFFFF` in order to match _any_ service type.
+* *MAY* have the _service instance ID_ part of its `ue_id` set to `0xFFFF` in order to match _any_ service instance. `0x0000` is the default instance ID used when there is only a single instance of a service.
```

Comment:
:+1:

Reply:
@stevenhartley 
Given that the semantics have been changed, the OFT spec item's revision needs to also be changed: `[.specitem,oft-sid="dsn~uri-pattern-matching~2",oft-needs="impl,utest"]`

This is necessary in order for OpenFastTrace to be able to detect that existing implementations no longer cover this specification item.